src/Sulu/Bundle/WebsiteBundle/Cache/Events.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\WebsiteBundle\Cache;
+
+/**
+ * Events related to clearing caches.
+ */
+final class Events
+{
+    /**
+     * Will be raised after http cache has been cleared.
+     */
+    const POST_CLEAR = 'sulu_website.cache.post_clear';
```

Comment:
I get that the `post` in the name would make more clear when it happens, but I think we have never used a notion like this in our code yet. Therefore I would prefer to rename it to `sulu_website.cache_clear`, in order to be consistent. Consistency makes the product easier to use :slightly_smiling_face:

Reply:
Modified it, and defined it as a class constant in the `CacheClearer` service.
Do you prefer to keep such constants in the related interface instead?

Reply:
I think until now we've always had a separate `Events` class, as you've had here... Can you again build it that way?

Reply:
I re-added the `Events` class.